amazon/ion/ioncmodule.c

```diff
@@ -1577,6 +1578,18 @@ PyObject* ionc_init_module(void) {
     _ionpydict_cls              = PyObject_GetAttrString(_simpletypes_module, "IonPyDict");
     _ionpydict_fromvalue        = PyObject_GetAttrString(_ionpydict_cls, "from_value");
 
+    // A temporary workaround to replace from_value() with class constructors
+    // TODO deprecate from_value later
+    _ionpyint_fromvalue = _ionpyint_cls;
+    _ionpynull_fromvalue = _ionpynull_cls;
+    _ionpyfloat_fromvalue = _ionpyfloat_cls;
+    _ionpydecimal_fromvalue = _ionpydecimal_cls;
+    _ionpytext_fromvalue = _ionpytext_cls;
+    _ionpylist_fromvalue = _ionpylist_cls;
+    _ionpydict_fromvalue = _ionpydict_cls;
+    _ionpybool_fromvalue = _ionpybool_cls;
+    _ionpysymbol_fromvalue = _ionpysymbol_cls;
```

Comment:
Should we remove the lines like
```
_ionpynull_fromvalue        = PyObject_GetAttrString(_ionpynull_cls, "from_value");
```
above, which these seem to overwrite?

Reply:
Yep, I added a new action item 2 to the next step list above. Having this methods overwrite here can switch between `from_value()` and constructor for easier benchmarking.